probe-rs/Cargo.toml

```diff
@@ -51,7 +52,7 @@ object = { version = "0.36", default-features = false, features = [
 ] }
 nusb = "0.1.12"
 futures-lite = { version = "2", default-features = false }
-async-io = "2"
+async-io.workspace = true
```

Comment:
Would you please move this to the other `workspace=true` deps?

Reply:
We should clean up deps in general.